recipes/fbpic/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.9.2' %}
+{% set name = 'FBPIC' %}
+{% set hash = '86523486b030e8a744c11ece2f0f2d7b47098f9954603fb83d8c89d94a454f86' %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ version }}.tar.gz'
+  url:
+    - 'https://github.com/{{ name|lower }}/{{ name|lower }}/archive/{{ version }}.tar.gz'
+  sha256: '{{ hash }}'
+
+build:
+  skip: True    # [win]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - pytest
+    - numba
+    - scipy
+    - h5py
+    - mkl
+    - mpi4py
+    - pyculib
+    - python-dateutil
+    - setuptools
+    - funcsigs
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - numba
+    - pytest
```

Comment:
Do you really needs ``pytest`` to run that?
Or is it necessary just for testing?